Cargo.toml

```diff
@@ -14,7 +14,7 @@ libc = "0.2.142"
 nix = { version = "0.26", default-features = false, features = ["signal"] }
 rustc-demangle = "0.1.23"
 spin_sleep = "1.1.1"
-wasmtime = ">= 15.0.0, < 23.0.0"
+wasmtime = ">= 15.0.0, < 28.0.0"
```

Comment:
I suggest we pin it. The only real consumer of this library is `function-runner`, which is always on a specific wasmtime version.

Reply:
It's a public crate so we can't know who other consumers may be. That said, there are no other crates on crates.io consuming this crate. So there may also be no other consumers at all.

Reply:
Even if there were, by specifying this dependency this way we're saying that this crate is compatible with that range of Wasmtime versions, which:  we're not guaranteeing or committing to gurantee in any shape or form, especially since upstream doesn't guarantee compatibility across major versions.